File: dayFour/afterBreak/python/taskTen.py
valid_sum_total = 0
valid_average = 0
valid_counter = 0
print("Enter your scores below: ")
for scores in range(1, 11):
    score = int(input())
    if score >= 0 and score <= 100:
        valid_sum_total = valid_sum_total + score
        valid_counter += 1
    else:
        print('Invalid Score')
valid_average = valid_sum_total/valid_counter
print("The average of valid scores is:", valid_average)

File: dayFour/afterBreak/java/TaskTen.java
import java.util.*;


public class TaskTen {
	static Scanner input = new Scanner(System.in);
	
	public static void main() {
	
	int validSum = 0;
	int validAverage = 0;
	
	System.out.println("Enter the scores below: ");
	
	int counter= 0;
	for(; counter < 10; counter++)
	  {
	    int score = input.nextInt();
	    if(score >= 0 && score <= 100)
	      {
	        validSum += score;
	      } else { System.out.println("Invalid score");}
	  }
	  
	  validAverage = validSum/counter;
	System.out.println("The average of even scores is: " + validAverage);
	}
}
